evalml/tests/component_tests/test_estimators.py

```diff
@@ -56,3 +60,25 @@ class MockEstimator(Estimator):
     mock_estimator.supported_problem_types = ['binary', 'multiclass']
     assert mock_estimator != MockEstimator()
     assert 'Mock Estimator' != mock_estimator
+
+
+@pytest.mark.parametrize("data_type", ['li', 'np', 'pd', 'ww'])
```

Comment:
@bchen1116 Great coverage, do we need to check if `predict` and `predict_proba` are appropriately covered as well?

Reply:
@ParthivNaresh I'll add in tests for those too. Thanks!